pySDC/projects/Resilience/fault_stats.py

```diff
@@ -1659,20 +1732,22 @@ def bar_plot_thing(
 
 def main():
     stats_analyser = FaultStats(
-        prob=run_Schroedinger,
+        prob=run_leaky_superconductor,
```

Comment:
here

Reply:
With this script I actually make the statistics for resilience so the problem here is just whatever I last checked. There is a test checking that the overall script works, which generates stats for van der Pol. I am not sure how to test that parameters work. I can ofc. test the stats generation for all problems, but its actually quite expensive. I think testing the parameters for all problems would be excessive.